javabuilder-authorizer/token_validator.rb

```diff
@@ -226,17 +247,17 @@ def user_usage(time_range_seconds)
     response
   end
 
-  def get_user_near_limit_detail(count, limit)
-    if count <= limit && count >= (limit - NEAR_LIMIT_BUFFER)
-      return {remaining: limit - count}
+  def get_user_near_limit_detail(count, limit, period, lockout_type)
+    if limit > 0 && count <= limit && count >= (limit - NEAR_LIMIT_BUFFER)
```

Comment:
minor: if we want to be super clear, maybe we can save 0 as a constant like "NO_LIMIT" or something, and check `limit != NO_LIMIT` here? The 0 is documented well enough in the template so maybe not necessary

Reply:
since this is slightly hacky, it doesn't hurt to be super clear. I'll update this